docs/reference/cluster-config.txt

```diff
@@ -379,256 +305,387 @@ The master supports the following configuration settings:
+                  tasks that do not specify a priority. Can be
+                  configured to 1 to 99 inclusively. Defaults to 42.
+
+         -  ``fitting_policy``: The scheduling policy to use when
+            assigning tasks to agents in the cluster. Defaults to
+            ``best``.
+
+            -  ``best``: The best-fit policy ensures that tasks will be
+               preferentially "packed" together on the smallest number
+               of agents.
+
+            -  ``worst``: The worst-fit policy ensures that tasks will
+               be placed on under-utilized agents.
+
+      -  ``default_cpu_resource_pool``: The default resource pool to use
+         for tasks that do not need GPUs. Defaults to ``default`` if no
+         resource pool is specified.
+
+      -  ``default_gpu_resource_pool``: The default resource pool to use
+         for tasks that need GPUs. Defaults to ``default`` if no
+         resource pool is specified.
+
+   -  ``type: kubernetes``: The ``kubernetes`` resource manager launches
+      tasks on a Kubernetes cluster. The Determined master must be
+      running within the Kubernetes cluster. When using the
+      ``kubernetes`` resource provider, we recommend deploying
+      Determined using the :ref:`Determined Helm Chart
+      <install-on-kubernetes>`. When installed via Helm, the
+      configuration settings below will be set automatically.
+
+      -  ``namespace``: The namespace where Determined will deploy Pods
+         and ConfigMaps.
+
+      -  ``max_slots_per_pod``: Each multi-GPU (distributed training)
+         task will be scheduled as a set of ``slots_per_task /
+         max_slots_per_pod`` separate pods, with each pod assigned up to
+         ``max_slots_per_pod`` GPUs. Distributed tasks with sizes that
+         are not divisible by ``max_slots_per_pod`` are never scheduled.
+         If you have a cluster of different size nodes, set
+         ``max_slots_per_pod`` to the greatest common divisor of all the
+         sizes. For example, if you have some nodes with 4 GPUs and
+         other nodes with 8 GPUs, set ``maxSlotsPerPod`` to ``4`` so
+         that all distributed experiments will launch with 4 GPUs per
+         pod (with two pods on 8-GPU nodes).
+
+      -  ``master_service_name``: The service account Determined uses to
+         interact with the Kubernetes API.
+
+-  ``resource_pools``: The resource pools to use to acquire resources.
+   Defaults to a resource pool with a name ``default``.
+
+   -  ``pool_name``: The name of the resource pool.
+
+   -  ``description``: The description of the resource pool.
+
+   -  ``max_cpu_containers_per_agent``: The maximum number of containers
+      that do not take any GPUs on each agent.
+
+   -  ``scheduler``: Specifies how Determined schedules tasks to agents.
+      The scheduler configuration on each resource pool will override
+      the global one.
+
+      -  ``type``: The scheduling policy to use when allocating
```

Comment:
There should be three types, `fair_share`, `priority`, and `round-robin`. The priority scheduler configuration should probably use the same fields as the global scheduler configuration.